web_src/less/_repository.less

```
@@ -861,6 +861,11 @@
           }
         }
 
+        /* Don't show the inline timeline avatar on desktop. */
```

```suggestion
        /* Don't show the mobile oriented avatar ".inline-timeline-avatar" on desktop. Desktop uses the avatar with class ".timeline-avatar" */
```

Well, I see the purpose now. Although it's a little strange to use 2 DOM elements for avatar, if it is a must, then it's fine.

---

Yeah, I haven't been able to find any other CSS solution to force it to be within the comment header. The placements of the timeline-avatar is not really suited for it.